.github/CODEOWNERS

```diff
@@ -31,13 +31,16 @@
 /src/promptflow-core/promptflow/storage/ @microsoft/promptflow-execution
 /src/promptflow-devkit/promptflow/_internal/ @microsoft/promptflow-execution
 /src/promptflow-devkit/promptflow/batch/ @microsoft/promptflow-execution
-/src/promptflow-devkit/promptflow/_proxy/ @microsoft/promptflow-execution
-
 /src/promptflow/tests/executor/ @microsoft/promptflow-execution
+
+/src/promptflow-devkit/promptflow/_proxy/ @microsoft/promptflow-execution @microsoft/promptflow-sdk
+
 /src/promptflow/tests/sdk_cli_test/ @microsoft/promptflow-sdk
 /src/promptflow/tests/sdk_cli_azure_test/ @microsoft/promptflow-sdk
 /src/promptflow/tests/sdk_cli_global_config_test/ @microsoft/promptflow-sdk
 /src/promptflow/tests/sdk_pfs_test/ @microsoft/promptflow-sdk
+/src/promptflow/tests/executor/_prompty_executor.py @microsoft/promptflow-sdk
+/src/promptflow/tests/executor/_script_executor.py @microsoft/promptflow-sdk
```

Comment:
Add @microsoft/promptflow-execution for this?

Reply:
Hmm, so only this line will take effect finally?